TestApp/ios/TestApp/AppDelegate.m

```diff
@@ -2,25 +2,25 @@
 // Licensed under the MIT License.
 
 /**
- * Copyright (c) 2015-present, Facebook, Inc.
- * All rights reserved.
+ * Copyright (c) Facebook, Inc. and its affiliates.
  *
- * This source code is licensed under the BSD-style license found in the
- * LICENSE file in the root directory of this source tree. An additional grant
- * of patent rights can be found in the PATENTS file in the same directory.
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
  */
 
 #import "AppDelegate.h"
-#import <AppCenterReactNative/AppCenterReactNative.h>
-#import <AppCenterReactNativeAuth/AppCenterReactNativeAuth.h>
-#import <AppCenterReactNativePush/AppCenterReactNativePush.h>
-#import <AppCenterReactNativeAnalytics/AppCenterReactNativeAnalytics.h>
-#import <AppCenterReactNativeCrashes/AppCenterReactNativeCrashes.h>
-#import <AppCenterReactNativeShared/AppCenterReactNativeShared.h>
 
 #import <React/RCTBridge.h>
 #import <React/RCTBundleURLProvider.h>
 #import <React/RCTRootView.h>
+
+#import <AppCenterReactNativeShared/AppCenterReactNativeShared.h>
+#import <AppCenterReactNative.h>
```

Comment:
#import "AppCenterReactNative.h" ?

Reply:
It's C/C++ convention. I believe the rule of thumb is to use angle brackets `<>` to refer to a header file owned by the library and quotation marks `""` to refer to a header file in the same app. That is what we did in the past and we should do the same here.

We verified `#import "AppCenterReactNative.h"` and `#import <AppCenterReactNative.h>` both work in this case.